--- eza-0.18.13/Cargo.toml	2024-04-20 20:57:41.178760171 +0800
+++ eza-0.18.13/Cargo.toml	2024-04-20 20:58:47.427618300 +0800
@@ -192,12 +192,5 @@
 [target."cfg(target_os = \"linux\")".dependencies.proc-mounts]
 version = "0.3"
 
-[target."cfg(target_os = \"windows\")".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_System_Console",
-    "Win32_Foundation",
-]
-
 [target."cfg(unix)".dependencies.uzers]
 version = "0.11.3"
