diff --git a/Cargo.toml b/Cargo.toml
index 5e90b437..7aac6223 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,11 +6,23 @@ categories = ["command-line-utilities"]
 edition = "2021"
 rust-version = "1.70.0"
 exclude = [
+  "/completions",
+  "/docs",
   "/devtools/*",
+  "/man",
+  "/snap",
   "/Justfile",
   "/Vagrantfile",
   "/screenshots.png",
   "/tests",
+  "/cliff.toml",
+  "/deb.asc",
+  "/deny.toml",
+  "/flake.*",
+  "/powertest.yaml",
+  "/rustfmt.toml",
+  "/rust-toolchain.toml",
+  "/treefmt.nix",
 ]
 readme = "README.md"
 homepage = "https://github.com/eza-community/eza"
@@ -102,32 +114,17 @@ proc-mounts = "0.3"
 [target.'cfg(unix)'.dependencies]
 uzers = "0.11.3"
 
-[target.'cfg(target_os = "windows")'.dependencies]
-windows-sys = { version = "0.52.0", features = [
-  "Win32_System_Console",
-  "Win32_Foundation",
-] }
-
 [build-dependencies]
 chrono = { version = "0.4.34", default-features = false, features = ["clock"] }
 
 [dev-dependencies]
-criterion = { version = "0.5.1", features = ["html_reports"] }
 trycmd = "0.15"
 
 [features]
 default = ["git"]
 git = ["git2"]
-vendored-openssl = ["git2/vendored-openssl"]
-vendored-libgit2 = ["git2/vendored-libgit2"]
-# Should only be used inside of flake.nix
-nix = []
-# Should only be used inside of flake.nix locally (not on CI)
-nix-local = []
-# Should only be used inside of flake.nix
 # Shouldn't ever be used in CI (slow!)
 powertest = []
-nix-generated = []
 
 # use LTO for smaller binaries (that take longer to build)
 [profile.release]
@@ -136,7 +133,3 @@ strip = true
 opt-level = 3
 codegen-units = 1
 panic = 'abort'
-
-[[bench]]
-name = "my_benchmark"
-harness = false
