--- eza-0.18.22/Cargo.toml	2024-07-18 21:09:36.117357014 +0800
+++ eza-0.18.22/Cargo.toml	2024-07-18 21:14:54.982516419 +0800
@@ -16,11 +16,23 @@
 version = "0.18.22"
 authors = ["Christina Sørensen <christina@cafkafk.com>"]
 exclude = [
+    "/completions",
+    "/docs",
     "/devtools/*",
+    "/man",
+    "/snap",
     "/Justfile",
     "/Vagrantfile",
     "/screenshots.png",
     "/tests",
+    "/cliff.toml",
+    "/deb.asc",
+    "/deny.toml",
+    "/flake.*",
+    "/powertest.yaml",
+    "/rustfmt.toml",
+    "/rust-toolchain.toml",
+    "/treefmt.nix",
 ]
 description = "A modern replacement for ls"
 homepage = "https://github.com/eza-community/eza"
@@ -88,10 +100,6 @@
 [[bin]]
 name = "eza"
 
-[[bench]]
-name = "my_benchmark"
-harness = false
-
 [dependencies.ansi-width]
 version = "0.1.0"
 
@@ -167,10 +175,6 @@
 [dependencies.zoneinfo_compiled]
 version = "0.5.1"
 
-[dev-dependencies.criterion]
-version = "0.5.1"
-features = ["html_reports"]
-
 [dev-dependencies.trycmd]
 version = "0.15"
 
@@ -182,12 +186,7 @@
 [features]
 default = ["git"]
 git = ["git2"]
-nix = []
-nix-generated = []
-nix-local = []
 powertest = []
-vendored-libgit2 = ["git2/vendored-libgit2"]
-vendored-openssl = ["git2/vendored-openssl"]
 
 [target."cfg(target_os = \"linux\")".dependencies.proc-mounts]
 version = "0.3"
