--- eza-0.20.15/Cargo.toml	1970-01-01 08:00:01.000000000 +0800
+++ eza-0.20.15/Cargo.toml	2025-01-04 02:33:17.439647287 +0800
@@ -16,18 +16,23 @@
 version = "0.20.15"
 authors = ["Christina Sørensen <christina@cafkafk.com>"]
 exclude = [
+    "/completions",
     "/docs/",
     "/devtools/",
+    "/man",
     "/snap/",
     "/tests/",
     "/.config/",
     "/.github/",
+    "/cliff.toml",
     "/deb.asc",
     "/deny.toml",
     "/flake.*",
     "/justfile",
     "/powertest.yaml",
+    "/rustfmt.toml",
     "/rust-toolchain.toml",
+    "/treefmt.nix",
     "/.envrc",
     "/.gitignore",
     "/.git-blame-ignore-revs",
@@ -99,10 +104,6 @@
 [[bin]]
 name = "eza"
 
-[[bench]]
-name = "my_benchmark"
-harness = false
-
 [dependencies.ansi-width]
 version = "0.1.0"
 
@@ -195,10 +196,6 @@
 [dependencies.zoneinfo_compiled]
 version = "0.5.1"
 
-[dev-dependencies.criterion]
-version = "0.5.1"
-features = ["html_reports"]
-
 [dev-dependencies.trycmd]
 version = "0.15"
 
@@ -210,22 +207,9 @@
 [features]
 default = ["git"]
 git = ["git2"]
-nix = []
-nix-generated = []
-nix-local = []
-powertest = []
-vendored-libgit2 = ["git2/vendored-libgit2"]
-vendored-openssl = ["git2/vendored-openssl"]
 
 [target."cfg(target_os = \"linux\")".dependencies.proc-mounts]
 version = "0.3"
 
-[target."cfg(target_os = \"windows\")".dependencies.windows-sys]
-version = "0.59.0"
-features = [
-    "Win32_System_Console",
-    "Win32_Foundation",
-]
-
 [target."cfg(unix)".dependencies.uzers]
 version = "0.12.1"
