--- eza-0.18.6/Cargo.toml	2024-03-06 21:44:57.181089586 +0800
+++ eza-0.18.6/Cargo.toml	2024-03-06 21:45:49.759766104 +0800
@@ -193,12 +193,5 @@
 [target."cfg(target_os = \"linux\")".dependencies.proc-mounts]
 version = "0.3"
 
-[target."cfg(target_os = \"windows\")".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_System_Console",
-    "Win32_Foundation",
-]
-
 [target."cfg(unix)".dependencies.uzers]
 version = "0.11.3"
